pkg/inject/report.go

```diff
@@ -20,6 +21,7 @@ const (
 	invalidInjectAnnotationWorkload      = "invalid_inject_annotation_at_workload"
 	invalidInjectAnnotationNamespace     = "invalid_inject_annotation_at_ns"
 	disabledAutomountServiceAccountToken = "disabled_automount_service_account_token_account"
+	UncompatbleNodeOS					 = "node_os_is_windows"
```

Comment:
Any reason why you've made this field public?

Reply:
honestly, no. Do you suggest I should leave it out?

Reply:
I think you might want to leave it as `unsupportedOS`. I don't see why that field should be exported.

Reply:
ya, I see what you mean, it's unnecessary to export it. Think it will be limited to windows only and that too will go away possibly. didn't think too much about it earlier. thanks!